internal/domain/instance.go

```diff
@@ -11,6 +11,9 @@ const (
 type Instance struct {
 	models.ObjectRoot
 
+	Name                            string
```

Comment:
we never use the Instance model, i'd only specify the state here

Reply:
I will remove this, if we need it this will come with the branch of @adlerhurst